gateway/src/main/java/discord4j/gateway/json/dispatch/InviteCreate.java

```diff
@@ -0,0 +1,79 @@
+package discord4j.gateway.json.dispatch;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import discord4j.common.jackson.UnsignedJson;
+import discord4j.common.json.UserResponse;
+
+public class InviteCreate implements Dispatch {
+
+    @JsonProperty("guild_id")
+    @UnsignedJson
+    private long guildId;
+    @JsonProperty("channel_id")
+    @UnsignedJson
+    private long channelId;
+    private String code;
+    @JsonProperty("created_at")
+    private String createdAt;
+    private int uses;
+    @JsonProperty("max_uses")
+    private int maxUses;
+    @JsonProperty("max_age")
+    private int maxAge;
+    private Boolean temporary;
+    //TODO: Inviter return null
+    @JsonUnwrapped
```

Comment:
I don't think this should be unwrapped?

Reply:
My error with the little copypaste for testing, removed and working.